tests/dateutils.rs

```diff
@@ -52,34 +52,68 @@ fn verify_against_date_command_local(path: &'static str, dt: NaiveDateTime) {
     }
 }
 
+/// path to Unix `date` command. Should work on most Linux and Unixes. Not the
+/// path for MacOS (/bin/date) which uses a different version of `date` with
+/// different arguments (so it won't run which is okay).
+/// for testing only
+#[allow(dead_code)]
+#[cfg(not(target_os = "aix"))]
+const DATE_PATH: &'static str = "/usr/bin/date";
+#[allow(dead_code)]
+#[cfg(target_os = "aix")]
+const DATE_PATH: &'static str = "/opt/freeware/bin/date";
+
+#[cfg(test)]
+/// test helper to sanity check the date command behaves as expected
+/// asserts the command succeeded
+fn assert_run_date_version() {
+    // note environment variable `LANG`
```

Comment:
Is this still useful when with https://github.com/chronotope/chrono/pull/1089?

Reply:
Yes, this check is unrelated to setting `LANG=c`.

We don't know precisely which `date` implementation is getting used (GNU, BSD, Solaris, etc.). So when it fails on some platform that is rarely used, this information will help confirm what's happening.

Reply:
Good argument.